interfaces/mount/entry.go

```diff
@@ -46,6 +46,25 @@ type Entry struct {
 	CheckPassNumber int
 }
 
+func sameStrings(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := 0; i < len(a); i++ {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+// EqualEntries checks if one entry is equal to another
+func EqualEntries(a, b *Entry) bool {
```

Comment:
another option would have been to make this a method of Entry:  func(a *Entry) Equal(b *Entry) bool